boards/hamilton/include/board.h

```diff
@@ -0,0 +1,166 @@
+#define AT86RF2XX_PARAM_SPI_CLK  SPI_CLK_5MHZ
+#define AT86RF2XX_PARAM_CS       GPIO_PIN(PB, 31)
+#define AT86RF2XX_PARAM_INT      GPIO_PIN(PB, 0)
+#define AT86RF2XX_PARAM_SLEEP    GPIO_PIN(PA, 20)
+#define AT86RF2XX_PARAM_RESET    GPIO_PIN(PB, 15)
+/** @} */
+
+/**
+ * @name    LED pin definitions and handlers
+ * @{
+ */
+#define LED_PIN                  GPIO_PIN(0, 19)
+
+#define LED_PORT                 PORT->Group[0]
+#define LED_MASK                 (1 << 19)
+
+#define LED_ON                   (LED_PORT.OUTCLR.reg = LED_MASK)
+#define LED_OFF                  (LED_PORT.OUTSET.reg = LED_MASK)
+#define LED_TOGGLE               (LED_PORT.OUTTGL.reg = LED_MASK)
+/** @} */
+
+/**
+ * @name    Button pin definitions
+ * @{
+ */
+#define BTN_PORT                 PORT->Group[0]
+#define BTN_PIN                  GPIO_PIN(0, 18)
+#define BTN_MODE                 GPIO_IN_PU
+/** @} */
+
+/**
+ * @name FXOS8700 configuration
+ * Note that another fxos8700 operation option, FXOS8700_USE_ACC_RAW_VALUES,
+ * need to be set according to the application purposes
+ * @{
+ */
+#define FXOS8700_PARAM_I2C                I2C_DEV(0)
+#define FXOS8700_PARAM_ADDR               (0x1E)
+#ifndef FXOS8700_PARAM_ACC_RANGE
+#define FXOS8700_PARAM_ACC_RANGE          FXOS8700_REG_XYZ_DATA_CFG_FS__8G
+#endif
+#ifndef FXOS8700_PARAM_RENEW_INTERVAL
+#define FXOS8700_PARAM_RENEW_INTERVAL     (1000000ul)
+#endif
+/** @} */
+
+/**
+ * @name HDC1080 configuration
+ * @{
+ */
+#define HDC1000_PARAM_I2C            I2C_DEV(0)
+#define HDC1000_PARAM_ADDR           (0x40)
+#ifndef HDC1000_PARAM_RES
+#define HDC1000_PARAM_RES            HDC1000_14BIT
+#endif
+#ifndef HDC1000_PARAM_RENEW_INTERVAL
+#define HDC1000_PARAM_RENEW_INTERVAL (1000000ul)
+#endif
+/** @} */
+
+/**
+ * @name EKMB (PIR motion sensor) configuration
+ * @{
+ */
+#define PIR_PARAM_GPIO          GPIO_PIN(PA, 6)
+#define PIR_PARAM_ACTIVE_HIGH   (1)
+/** @} */
+
+/**
+ * @name PULSE_COUNTER configuration
+ * @{
+ */
+#define PULSE_COUNTER_GPIO          BTN_PIN
+#define PULSE_COUNTER_GPIO_FLANK    GPIO_FALLING
+/** @} */
+
+/**
+ * @name TMP006 configuration
+ * Note that two other tmp006 operation options, TMP006_USE_LOW_POWER and
+ * TMP006_USE_RAW_VALUES, need to be set according to the application purpose
+ * @{
+ */
+#define TMP006_PARAM_I2C            I2C_DEV(0)
+#define TMP006_PARAM_ADDR           (0x44)
+#define TMP006_PARAM_RATE           TMP006_CONFIG_CR_AS2
+/** @} */
+
+/**
+ * @name ToDo: APDS9007 configuration
+ * @{
+ */
+/** @} */
+
+/**
+ * @name Factory-generated parameters
+ * @{
+ */
+extern const uint64_t* const fb_sentinel;
+extern const uint64_t* const fb_flashed_time;
+extern const uint8_t*  const fb_eui64;
+extern const uint16_t* const fb_device_id;
+extern const uint64_t* const fb_designator;
+extern const uint8_t*  const fb_aes128_key;
+extern const uint8_t*  const fb_25519_pub;
+extern const uint8_t*  const fb_25519_priv;
+#define FB_SENTINEL_VALUE 0x27c83f60f6b6e7c8
```

Comment:
What is this value for ?

Reply:
@immesys, please answer to this question.